CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v17.17.0
```

Comment:
Did you pull the master branch? I thought https://github.com/kubernetes-client/python/pull/1455 would be included

Reply:
No will pull it with `git pull -X theirs upstream master`.